plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/status-popover.tsx

```diff
@@ -14,35 +14,78 @@ export default function StatusPopover( props: {
 	text: string;
 	level: StatusLevel;
 	explanation: string | JSX.Element;
+	explanationOnHover?: boolean;
 } ) {
-	const [ isVisible, setIsVisible ] = useState( false );
+	const [ isHovered, setIsHovered ] = useState( false );
+	const [ isClicked, setIsClicked ] = useState( false );
+	const hoverTimeoutId = useRef< null | NodeJS.Timeout >( null );
+
+	useEffect( () => {
+		return () => {
+			if ( hoverTimeoutId.current ) {
+				clearTimeout( hoverTimeoutId.current );
+			}
+		};
+	}, [] );
+
+	const startHover = () => {
+		if ( ! props.explanationOnHover ) {
+			return;
+		}
+
+		if ( hoverTimeoutId.current ) {
+			clearTimeout( hoverTimeoutId.current );
+		}
+
+		setIsHovered( true );
+	};
+
+	const endHover = () => {
+		if ( ! props.explanationOnHover ) {
+			return;
+		}
+
+		if ( hoverTimeoutId.current ) {
+			clearTimeout( hoverTimeoutId.current );
+		}
+
+		// Add a small delay in case user hovers from the button to the popover.
+		// In such a case we don't want to hide the popover.
+		hoverTimeoutId.current = setTimeout( () => {
+			setIsHovered( false );
+		}, 350 );
+	};
 
 	function shouldShowExplanation() {
 		if ( props.explanation === '' ) {
 			return false;
 		}
 
-		return isVisible;
+		return isClicked || ( props.explanationOnHover && isHovered );
 	}
 
 	return (
 		<button
-			onClick={ () => {
-				setIsVisible( ! isVisible );
-			} }
-			className={
-				'woocommerce-marketplace__my-subscriptions__product-status' +
-				' ' +
+			onClick={ () => setIsClicked( ! isClicked ) }
+			onMouseOver={ startHover }
+			onFocus={ startHover }
+			onMouseOut={ endHover }
+			onBlur={ endHover }
+			className={ clsx(
+				'woocommerce-marketplace__my-subscriptions__product-status',
 				`woocommerce-marketplace__my-subscriptions__product-status--${ props.level }`
-			}
+			) }
 		>
-			<Icon icon={ info } size={ 16 } />
 			{ props.text }
 			{ shouldShowExplanation() && (
 				<Popover
```

Comment:
The popover content isn't being read by screen readers, so this info is completely invisible for non-sighted users. It would be good if we can improve this as well, though it might make sense to handle as a separate task. It also depends on whether this is an issue with our code, or with the component itself, which is part of WP Core...